includes/admin/class-sensei-learner-management.php

```diff
@@ -437,7 +450,19 @@ public function edit_date_started() {
 			exit( '' );
 		}
 
-		$updated = update_comment_meta( $comment_id, 'start', $mysql_date, $date_started );
+		$updated = (bool) update_comment_meta( $comment_id, 'start', $mysql_date, $date_started );
+
+		/**
+		 * Filter sensei_learners_default_columns
```

Comment:
This is the doc of another filter.

Reply:
Fixed in 3235c8b4334c24c92db8bc0e34ae27d0c29f31ac